data/json/snippets/missing_posters.json

```diff
@@ -0,0 +1,226 @@
+      },
+      {
+        "id": "maine_coon",
+        "text": "LOST CAT \n\n - ADULT BLACK & BROWN MALE MAINE COON CAT\n - WHITE ON NECK AND MUZZLE \n - WEARS REFLECTIVE GREEN COLLAR \n\n[Photograph: a large cat with a dense coat of fur sleeping in a plastic lawn chair.  He looks uncomfortable.] \n\nLoki from Shutesbury, MA. \n\nTAKE A PIC IF YOU CAN + PLEASE DON'T CHASE \n617-460-9958"
+      },
+      {
+        "id": "siamese",
+        "text": "MISSING CAT ALERT \n\n781-669-2647 \n\n[Photograph: a slender white cat with a grayish face peeking out a small cardboard box.] \n\nThis adorable Siamese cat is called COCOA. \nHe is new to the area and escaped out of the door.  He is microchipped and has a name tag on his collar.  He is very nervous around strangers and may flee if he thinks you are going to catch him. \n\nIf you see him or think you have seen a purrson like him, please call me ASAP.  We love him and miss him dearly."
+      },
+      {
+        "id": "persian",
+        "text": "!! HELP ME !! \nMy name is Pippa and I am lost! \n\n[Photograph: a long-haired cat with a round face and short muzzle sitting on a floral carpet.] \n\nI'm getting on with age, so I don't remember how to get home.  I am a (traditional) Persian longhair with silky white fur and brownish eyes.  I was wearing a red collar with my name tag on it. \n\nPLEASE CALL (413)347-1688 IF YOU SEE ME \n(AND TAKE A PICTURE IF YOU CAN)"
+      },
+      {
+        "id": "bengal",
+        "text": "MISSING CAT \n\n[Photograph: a shorthair cat with rosette markings training its claws on a scratching post.] \n\nCheddar is a male 3.5 y/o brown Bengal cat who went missing on April 8.  She was last seen wearing a striped collar. \n\nIF YOU SEE HIM, CONTACT US IMMEDIATELY: 978-860-3760 \n\n YOU CAN HELP! \nPlease photograph this poster and share it on social media."
+      },
+      {
+        "id": "devon_rex",
+        "text": "DEXTER IS LOST! \nAGAIN!!!  He was scared off by the neighbor's dogs on 6th May & doesn't know how to get home… \n…he's not very clever at finding his way back! \nHe's a 3 y/o male Devon Rex cat with soft gray fur and big blue eyes. \nHe's chipped and should be wearing a yellow collar with a tag that has his name & my phone number on it. \n\n[Photograph: a short-haired gray cat sitting next to four similar-looking felines.  Dexter is circled in and has a bright red arrow pointing at him.] \n\nIF YOU SEE MY DEXTER, PLEASE DIAL 978-615-8152! \n\n!!!REWARD OFFERED!!!"
+      },
+      {
+        "id": "sphynx",
+        "text": "ATTENTION \nMISSING CAT \n\n[Photograph: a hairless, wrinkled cat with lemon-shaped eyes wearing a striped T-shirt.] \n\nRAMSES went missing from the car park at Companion Care Vets in Oakland on 30th March. \nHe is a Canadian Sphynx with a gray spot covering its stomach. \nShe is wearing a striped T-shirt and a pink collar. \n\nIf found, please contact me at: iFoundYourCat@gmail.com"
```

Comment:
I would use an imaginary mail provider and TLD, just because there could be folks who actually try to write to it. E.g iFoundYourCat@adrex-online.edge

Yes, adrex is xedra but scrambled because I could not make anything else up.

Reply:
I went ahead and made a working email address for Ramses the Cat.  You are free to contact me in case you found him.